cloud-controller-manager/instances.go

```diff
@@ -117,8 +117,19 @@ func (s *InstanceClient) filterOutByLabel(nodes []*v1.Node, labels string) ([]*v
 	return result, nil
 }
 
-// Use '.' to separate providerID which looks like 'cn-hangzhou.i-v98dklsmnxkkgiiil7'. The format of "REGION.NODEID"
+// providerID
+// 1) the id of the instance in the alicloud API. Use '.' to separate providerID which looks like 'cn-hangzhou.i-v98dklsmnxkkgiiil7'. The format of "REGION.NODEID"
+// 2) the id for an instance in the kubernetes API, which has 'alicloud://' prefix. e.g. alicloud://cn-hangzhou.i-v98dklsmnxkkgiiil7
```

Comment:
We'd better make it like `alicloud:///cn-hangzhou/i-v98dklsmnxkkgiiil7`, which is consistent with the standard format `<cloudProvider>://<optional>/<segments>/<provider id>`.

Reply:
`provider_id` in alicloud is like 'cn-hangzhou.i-v98dklsmnxkkgiiil7'. The format is "REGION.NODEID"

Reply:
OK. Let's keep the original design `alicloud://cn-hangzhou.i-v98dklsmnxkkgiiil7` for now. It seems that it can be parsed with
https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/noderefutil/providerid.go#L47